src/collective/fingerpointing/browser/logview.pt

```diff
@@ -21,9 +21,82 @@
     </metal:title>
 
     <metal:content-core metal:fill-slot="content-core">
-      <pre tal:condition="view/available" tal:content="view/get_audit_log" />
+      <tal:block tal:condition="python:view.available"
+                 tal:define="dummy python:view.build_audit_log()">
+        <form method="get"
+              action=""
+              tal:attributes="action python:view.base_url">
+          <div class="field">
+            <input type="hidden"
+                   name="size"
+                   value=""
+                   tal:attributes="value python:view.size" />
+            <input type="text"
+                   name="query"
+                   value=""
+                   class="searchButton"
+                   placeholder="Enter a search term."
+                   i18n:attributes="placeholder"
+                   tal:attributes="value python:view.query" />
+          </div>
+          <div class="formControls">
```

Comment:
I think this `<div>` is useless.

Reply:
I just copied it as-is from some existing Plone form.